### ISSUE TITLE
No Overload method for 'PullAsync' in AzureService.cs requires 0 arguments

### ISSUE THREAD

**Issue by pmace** [xamarin/VervetaCRM#1]
.PullAsync calls missing queryID and query.
project won't compile.

**Comment by marcvthooft** [comment]
@RoDo777 I also use VervetaCRM and now stumble on the "arg cannot be null, Parameter name: image" issue. Did you find a solution?

**Comment by RoDo777** [comment]
@marcvthooft No, I didnt invest more time in this. @stevenyix said in a previous comment they will check this in the next update.
